



async def get_candidate_memories(candidate_id:str,limit:int=10):
    try:
        memories = ""
        return memories
    except Exception as e:
        print(f"Error at get_candidate_memories :-: {e}")